lib/hutch/config.rb

```diff
@@ -13,6 +13,7 @@ def self.initialize(params={})
         mq_host: 'localhost',
         mq_port: 5672,
         mq_exchange: 'hutch',  # TODO: should this be required?
+        mq_exchange_options: {},
```

Comment:
No need for this since you are already doing the hash merge thing in broker.rb above.